PyInstaller/cli/__init__.py

```diff
@@ -0,0 +1,74 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2021, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+# -----------------------------------------------------------------------------
+
+import click
+
+from PyInstaller import __version__, compat
+from . import opts
+
+CONTEXT_SETTINGS = {
+    'help_option_names': ['-h', '--help'],
+}
+
+
+@click.group(context_settings=CONTEXT_SETTINGS)
+@click.version_option(__version__)
+@click.pass_context
+def cli(_ctx: click.Context):
+    """Main Help Text
+    """
+
+    # This function is run whenever one of the subcommands is called, so we can add global-code here.
+    compat.check_requirements()
+
+
+@cli.command('build')
+@opts.add_logging_options
+@opts.add_build_options
+@opts.add_makespec_options
+@click.argument(
+    'file', metavar='SCRIPTNAME',
```

Comment:
Does this mean we are eliminating the option to have multiple scripts in a  single executable (presumably ran one after another)?

Reply:
Nope, you'll just have to write the specfile yourself if you want multiple scripts.

Reply:
To be honest, if I were designing PyInstaller's API from scratch, I think I'd allow multiple scripts but make it equivalent to using `Merge()` instead of concatenating them but that's a lot of floorboards to rip up. I can't think why anyone would want to concatenate scripts like the current API does so I wouldn't object to removing that completely.